lib/pr_query.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// lib/queries/*.gql file names
+const PR_QUERY = 'PRQuery';
+
+class PRQuery {
+  /**
+   * @param {Object} argv
+   * @param {Object} cli
+   * @param {Object} request
+   */
+  constructor(argv, cli, request) {
+    const { assignee, owner, repo } = argv;
```

Comment:
I think it's probably enough to just pass these directly here instead of through `argv` - this is very far away from the command handler now and `argv` isn't too meaningful here.